docs/src/types/vectors.md

```diff
@@ -0,0 +1,183 @@
+# Vectors
+
+## Passing in vectors
```

Comment:
I think we're meant to add an empty line after titles. Otherwise, the Markdown linter will complain.

Reply:
I wasn't aware our CI had a markdown linter. Added the newlines.